mod/wikirate/lib/open_corporates/api.rb

```diff
@@ -34,11 +34,6 @@ def fetch_jurisdictions
       # @return the full json response converted to a hash
       def fetch *query_args
         JSON.parse json_response(*query_args)
```

Comment:
Unlikely that if we get a response that it's not json. Airbrake will tell us if I'm wrong